extension/src/style.pug

```diff
@@ -7,7 +7,7 @@ style.
     overflow: hidden;
     height: 100%;
     width: 100%;
-    min-width: 350px;
+    min-width: 720px;
```

Comment:
maybe a 760px would be better, like it used to be?

Reply:
To be clear, the popup used to size to content which was almost always bigger than `min-width`. I suspect a change in Chrome 116 makes it so it no longer resizes to fit the content.